lsp-ui-sideline.el

```diff
@@ -318,14 +318,28 @@ CURRENT is non-nil when the point is on the symbol."
         (when (overlay-get ov 'current)
           (lsp-ui-sideline--toggle-current ov nil))))))
 
+(defun lsp-ui-sideline--split-long-lines (lines)
+  "Show diagnostics on the current line."
+  (mapcan (lambda (line)
+            (let* ((max-length 100))
```

Comment:
Maybe we could consider making the ` 100` a variable? It depends on the available screen space, so some users may want to tweak it.